README.rst

```diff
@@ -10,7 +10,7 @@ Development
 PySWMM Wiki
 	https://github.com/OpenWaterAnalytics/pyswmm/wiki/
 Cite our Paper
-	https://doi.org/10.5281/zenodo.3751574
+	https://doi.org/10.21105/joss.02292
```

Comment:
I’d suggest going as far as adding the bibtex format and the string clean version here

Reply:
I added the full citation - not sure what you mean about the clean string version though. Feel free to tweak!